server/views/review/includes/curfewAddress.pug

```diff
@@ -76,7 +77,10 @@ if licenceStatus.stage !== 'ELIGIBILITY' && !addressReviewNeeded
   if !offenderIsMainOccupier
     div.pure-g.borderBottomLight.midPaddingTopBottom
       div.pure-u-1-2
-        span.pure-u-3-4 Does the main occupier consent to HDC?
+        if curfewAddressReviewVersion1
+          span.pure-u-3-4 Does the main occupier consent to HDC?
+        else 
+          span.pure-u-3-4 Have you spoken to the main occupier and do they consent to HDC?
```

Comment:
Again I'd be tempted to have 2 separate template files for each version 😅

Reply:
as per previous really

Reply:
ok